### DISCUSSION TITLE
Zoraxy - Proxmox VE Helper Script

### DISCUSSION THREAD

**Discussion by DocDrydenn** [tobychui/zoraxy#37] [General]
Let me start by saying that I know installing and running Zoraxy is really simple and easy. That said, it never hurts to have some great automation involved, so I figured I'd share it here.

Anyway, @tteck was kind enough to add a Zoraxy LXC script to his extensive [Proxmox VE Helper Scripts](https://tteck.github.io/Proxmox/) (it's located under the `Server - Networking` sub section).

I've tested it out and all seems to be working very well. So if you're a Proxmox user and are looking for a simple and easy way to spin up an LXC running Zoraxy, then give his repo/script a look.

**Comment by VinterSolen** [comment]
The Proxmox VE Helper Scripts has moved to https://community-scripts.github.io/ProxmoxVE/scripts?id=zoraxy


Just an FYI. (It also says that on the first post link)
